packages/shipjs/src/step/finished.js

```diff
@@ -1,3 +1,21 @@
+import chalk from 'chalk';
 import runStep from './runStep';
+import { print } from '../util';
 
-export default () => runStep({ title: 'All Finished.' }, () => {});
+export default ({ baseBranch, stagingBranch, pullRequestUrl, dryRun }) =>
+  runStep({ title: 'All Finished.' }, () => {
+    const prURL = dryRun
+      ? chalk.gray('(Because of --dry-run, pull-request url has been omitted)')
+      : pullRequestUrl;
+    print(`
+You are currently on ${chalk.green.bold(
+      stagingBranch
+    )} branch, the purpose of which is to make changes to release materials including CHANGELOG.md, and continue to prepare the next release.
```

Comment:
```suggestion
    )} branch. You can make changes to the release materials including CHANGELOG.md, and continue to prepare the next release.
```

What do you think?

Reply:
Seems better!

```suggestion
    )} branch.
You can make changes to the release materials including CHANGELOG.md, and continue to prepare the next release.
```

How about adding line break?

Reply:
Good 👍